packages/mainmenu-extension/schema/plugin.json

```diff
@@ -270,22 +270,22 @@
     {
       "command": "kernelmenu:interrupt",
       "keys": ["I", "I"],
-      "selector": "[data-jp-kernel-user] :focus:not(:read-write)"
+      "selector": "[data-jp-kernel-user]:not(.jp-mod-readWrite) :focus:not(:read-write)"
```

Comment:
Is there a reason to keep the `:not(read-write)` here ?

Reply:
Yes, I added it back because extensions may have  elements which are kernel runners but would not have time to implement `.jp-mod-readWrite`. Basically out of abundance of caution to not break anything in the wild.